src/fidesops/service/connectors/saas_connector.py

```diff
@@ -117,8 +118,14 @@ def __init__(self, configuration: ConnectionConfig):
         self.collection_name = None
 
     def query_config(self, node: TraversalNode) -> SaaSQueryConfig:
-        """Returns the query config for a SaaS connector"""
-        return SaaSQueryConfig(node, self.endpoints, self.secrets)
+        """Returns the query config for a given node"""
+        collection_name = node.address.collection
+        configured_masking_request = self.get_masking_request_from_config(
+            collection_name
+        )
+        return SaaSQueryConfig(
+            node, self.endpoints, self.secrets, configured_masking_request
+        )
```

Comment:
Setting the masking request here when we instantiate the `SaaSQueryConfig` so it can be used both in `SaasConnector.mask_data` as well as `SaaSQueryConfig.generate_update_stmt`